lib/models/config.coffee

```diff
@@ -24,12 +25,13 @@ request = require('resin-request')
 #		console.log(config)
 ###
 exports.getAll = (callback) ->
-	request.request
-		method: 'GET'
-		url: '/config'
-	, (error, response, config) ->
-		return callback(error) if error?
-		return callback(null, config)
+	Promise.fromNode (callback) ->
```

Comment:
It would be nicer to `Promise.promisifyAll(request)` and then use the async versions